### ISSUE TITLE
Attachments Not Showing

### ISSUE THREAD

**Issue by mookerific** [nylas/N1#2537]
N1 on Win10:

N1 intermittently fails to show attachments to e-mail in which there are most definitely attachments, both in the e-mail list (no paperclip) and also when an e-mail is open (no attachments available to click on).

**Comment by bengotow** [comment]
Hey! Thanks for reporting this - do you know offhand if the missing attachments have anything in common, like a file type? Were they sent from the mail client? I'm afraid we won't be able to fix this without more specifics.

**Comment by jackiehluo** [comment]
Another incidence: https://nylas.zendesk.com/agent/tickets/3690

**Comment by mookerific** [comment]
I can't open this, but I imagine that's because this was intended for someone
else at Nylas.  
  
Sent from [Nylas N1](https://link.nylas.com/link/ejkuvebexf2g9r7yim6hd14x2
/local-
cd6c98a8-278e/0?redirect=https%3A%2F%2Fnylas.com%2Fn1%3Fref%3Dn1&r=cmVwbHkrMDEyNzRiZTljM2I2YzFjYTZjMTVmMzZmNzFiZTEzMWQ0YTBlNDJkYzYwMmIyNTUxOTJjZjAwMDAwMDAxMTM5NTMxNTM5MmExNjljZTA5YmFmNTZiQHJlcGx5LmdpdGh1Yi5jb20=),
the extensible, open source mail client.  
![](https://link.nylas.com/open/ejkuvebexf2g9r7yim6hd14x2/local-
cd6c98a8-278e?r=cmVwbHkrMDEyNzRiZTljM2I2YzFjYTZjMTVmMzZmNzFiZTEzMWQ0YTBlNDJkYzYwMmIyNTUxOTJjZjAwMDAwMDAxMTM5NTMxNTM5MmExNjljZTA5YmFmNTZiQHJlcGx5LmdpdGh1Yi5jb20=)

On Jul 6 2016, at 4:51 pm, Jackie Luo &lt;notifications@github.com&gt; wrote:  

> Another incidence:
[https://nylas.zendesk.com/agent/tickets/3690](https://nylas.zendesk.com/agent/tickets/3690&r=cmVwbHkrMDEyNzRiZTljM2I2YzFjYTZjMTVmMzZmNzFiZTEzMWQ0YTBlNDJkYzYwMmIyNTUxOTJjZjAwMDAwMDAxMTM5NTMxNTM5MmExNjljZTA5YmFmNTZiQHJlcGx5LmdpdGh1Yi5jb20=)

>

> —  
> You are receiving this because you authored the thread.  
> Reply to this email directly, [view it on
GitHub](https://github.com/nylas/N1/issues/2537#issuecomment-230902450&r=cmVwbHkrMDEyNzRiZTljM2I2YzFjYTZjMTVmMzZmNzFiZTEzMWQ0YTBlNDJkYzYwMmIyNTUxOTJjZjAwMDAwMDAxMTM5NTMxNTM5MmExNjljZTA5YmFmNTZiQHJlcGx5LmdpdGh1Yi5jb20=),
or [mute the
thread](https://github.com/notifications/unsubscribe/ASdL6cUMtdTP2mMozKEPO0w2n7kG1Gzjks5qTBVTgaJpZM4JCgib&r=cmVwbHkrMDEyNzRiZTljM2I2YzFjYTZjMTVmMzZmNzFiZTEzMWQ0YTBlNDJkYzYwMmIyNTUxOTJjZjAwMDAwMDAxMTM5NTMxNTM5MmExNjljZTA5YmFmNTZiQHJlcGx5LmdpdGh1Yi5jb20=).![](https://github.com/notifications/beacon/ASdL6ba9YJuk1YpMc4je6cHrBtNfCl4Hks5qTBVTgaJpZM4JCgib.gif)

**Comment by bengotow** [comment]
Another incidence of this issue was reported today: https://nylas.zendesk.com/agent/tickets/6158

**Comment by vsanchez** [comment]
I can confirm this also. To add to report in ticket #2336 about the umlauts, it happens when the file name has an acute accent (as used in spanish, áéíóú)  when sent from an Outlook 386 account to another.

**Comment by vsanchez** [comment]
Hi, 

The problem is easy to replicate and very bothersome. It happens when an attachement has a file name with some special characters (in my case when it has an accented character for example á, é etc) and the mail is sent from an Outlook 365 to another Outlook 365 account. If it sent to an Gmail account then there is no problem... do you have a test Outlook 365 account I can send an attachment to test??

**Comment by khairulashraff** [comment]
Hello,

I recently ran in to a similar issue although possibly of a different cause.

The missing attachment is a PDF with Content-Disposition of inline. When I view the email from my GMail inbox, it's missing the paperclip however the attachment appears just fine in the email itself.

Nylas unfortunately doesn't see it and completely misses it.

**Comment by jbaf** [comment]
I ran into the same problem.
When it comes to a PDF attached as an inline element of the e-mail (like  MacOS app **Mail** does), Nylas Mail does not show the attachment.
If the PDF is integrated as a true attachement by the sender (like in Gmail, on a browser), it shows properly once received in Nylas Mail.

Could you fix this problem, please ?

**Comment by alexeyegorov** [comment]
I agree with @jbaf . I am using Nylas Mail since yesterday and already missing two attachments: both are PDF files with no special characters in the names. As both mails come from other colleges with macbooks, I assume this could possibly be some issue with Mail.app attachments! I hope you can fix it fast, so that we can enjoy your great app!

**Comment by ccpost** [comment]
I think I'm having the same problem over in #3429, but the inline attachment also seems to make Nylas Mail eat any body text after the inline attachment as well.

**Comment by cmoro-deusto** [comment]
I'm having the same problem, inline attachments don't show up. 

I'm using **v2.0.15** on linux:

```
Linux moria 4.10.0-19-generic #21-Ubuntu SMP Thu Apr 6 17:04:57 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=17.04
DISTRIB_CODENAME=zesty
DISTRIB_DESCRIPTION="Ubuntu 17.04"

```

**Comment by traf** [comment]
Having the same problem on Mac, version 2.0.16. No attachments on any new email show up for me.

**Comment by jstejada** [comment]
thank we're looking into this

**Comment by b0123498765** [comment]
Also having the same problem, PDF file, no special characters

**Comment by alexeyegorov** [comment]
Two weeks earlier I still had the same issue (as reported on 2nd February). I switched my mail program to Airmail, but as far as this issue is fixed I would definitely give nylas mail a second try.

**Comment by jbaf** [comment]
In my opinion, this issue is the most problematic one I encountered so far.
Always having to switch back to Mac Mail app when I receive a document with an inline PDF attached is quite anoying, as I get a lot of theses from my boss and from clients.
I see Nylas Mail as a very good alternative to Mac Mail, but should this problem last, I think I would pass on Nylas once again...

**Comment by jakedeg** [comment]
Yup. Same issue here and same issue as https://github.com/nylas/nylas-mail/issues/3429. No quoted text or attachment showing up. The attachment is a pdf with a standard ASCII filename. This is a serious enough issue to make me stop using Nylas. :(

**Comment by alexandersdad** [comment]
Darn it. I thought maybe I was having an isolated issue with attachments, but by the looks of things this is pretty widespread. Like @jakedeg I *want* to use Nylas, but, well, attachments not showing up is sort of a big deal.  

Win 10, v1703, x64

**Comment by six7** [comment]
Same here, this is the main problem stopping me from using Nylas.

**Comment by narratorben** [comment]
I have the same problem here.. I have to log onto GMail web interface to see the attachment. It seems to happen mostly from people who I know are using Mac Mail but there may be others.

**Comment by owain68** [comment]
This is also happening to me so I am having to go back to using Apple Mail.

**Comment by EliBon** [comment]
I've also encountered this issue. So far its only one sender, who uses mac mail, sending me PDF attachments that  just dont show up. When he zipped the file and resent it, I got the attachment no problem.

**Comment by mollierobbert** [comment]
Same here. PDF and images not showing up for a specific forwarded message, while the Gmail web client is showing it.

**Comment by munchee13** [comment]
Same here... attachments not showing if PDF on Mac.

**Comment by ephoras** [comment]
same here, pdfs not showing up at all :(

**Comment by goncalotomas** [comment]
Same here, using Nylas Mail 2.0.32 on MacOS Sierra.

**Comment by claudioruiz** [comment]
Same here

**Comment by chipbuster** [comment]
Same, Nylas mail 2.0.32 on Windows 10. I'll edit this if I can reproduce the behavior on Linux as well.

Interestingly, it doesn't appear to be any one sender client causing it--I'm getting it in mail sent from MacOS Mail and Sympa, and I suspect a few other clients as well.

**Comment by j-rahman** [comment]
Confirmed with Nylas Mail 2.0.32 on Xubuntu 16.04 (Linux). Why hasn't it been fixed since a year?

**Comment by thebenp** [comment]
Ditto, Nylas Mail 2.0.32 on Centos 7.

**Comment by gnpappas** [comment]
No PDFs are showing in emails. Semi-dealbreaker.

Windows 10 // Nylas 2.0.32

**Comment by danharms** [comment]
Unable to view PDFs on macOS Sierra (10.12.5) / Nylas Mail 2.0.32

**Comment by Alsciende** [comment]
Same here. Inline PDF doesn't show up.

```
Content-Type: multipart/mixed; boundary="Apple-Mail=_7A06A829-4A38-4613-A2F5-788C8A4D513E"
Mime-Version: 1.0 (Mac OS X Mail 10.3 \(3273\))
X-Mailer: Apple Mail (2.3273)

--Apple-Mail=_7A06A829-4A38-4613-A2F5-788C8A4D513E
Content-Transfer-Encoding: 7bit
Content-Type: text/plain; charset=us-ascii

Some text

--Apple-Mail=_7A06A829-4A38-4613-A2F5-788C8A4D513E
Content-Disposition: inline; filename=DDDViteFait.pdf
Content-Type: application/pdf; x-unix-mode=0644; name="DDDViteFait.pdf"
Content-Transfer-Encoding: base64


--Apple-Mail=_7A06A829-4A38-4613-A2F5-788C8A4D513E
Content-Transfer-Encoding: quoted-printable
Content-Type: text/plain; charset=utf-8

More text

--Apple-Mail=_7A06A829-4A38-4613-A2F5-788C8A4D513E--
```

**Comment by nicholasbutlin** [comment]
Also having this problem. 

Sent email from iOS mail on iPhone with two photo attachments. 
Email had no subject. 

Not showing any attachment in Nylas mail on linux, 2.0.32

**Comment by MisterTimn** [comment]
Having the exact same issue on Ubuntu 17.04 and Windows 10 (Nylas Mail 2.0.32)

Fix would be real nice 👍

**Comment by cmoro-deusto** [comment]
They have just announced that they are discontinuing Nylas Mail development. Check the URL guys, there are some forks that will keep on working on this.

https://www.nylas.com/blog/sunsetting-nylas-mail-development

Thanks to the Nylas team for their work and for open sourcing this!

**Comment by claudiorrrr** [comment]
check https://github.com/nylas-mail-lives/nylas-mail, the fork with the largest community of programmers working on

**Comment by AndrewKiri** [comment]
Having the issue as well on Ubuntu 16.04 LTS / Nylas v.2.0.32 with email attachments being invisible, which is sent from macOS mail app

**Comment by muzsen** [comment]
Same issue--no attachments showing in Nylas (though visible in web interface). PDF this time; will follow for other formats.

**Comment by tjerk-g** [comment]
I also don't see attachments in Nylas Mail 2.0.32. It was a PDF with a normal name including spaces (it's a CV, so I can't give the specifics)

**Comment by carlotrimarchi** [comment]
Just happened to me as well: inline attached pdf didn't show up
